src/pages/categoryCombos/Edit.tsx

```diff
@@ -11,7 +11,7 @@ const fieldFilters = [
     ...DEFAULT_FIELD_FILTERS,
     'name',
     'code',
-    'categories[id,displayName]',
+    'categories[id,displayName,categoryOptions~size~rename(categoryOptionsSize)],',
```

Comment:
wow. no idea you could do that!

Reply:
[Field transformers!](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/metadata.html#webapi_field_transformers)